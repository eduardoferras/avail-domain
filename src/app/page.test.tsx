import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import Page from './page'

describe('Page', () => {
	it('renders a heading', () => {
		render(<Page />)

		const heading = screen.getByRole('heading', { level: 1 })

		expect(heading).toBeInTheDocument()
	})

	it('should be rendered correctly', () => {
		const { container } = render(<Page />)

		expect(container).toMatchSnapshot()
	})
})
