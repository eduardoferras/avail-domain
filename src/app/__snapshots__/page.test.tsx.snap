// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page should be rendered correctly 1`] = `
<div>
  <main
    class="container"
  >
    <section
      class="Upload"
    >
      <h1
        class="Upload__title"
      >
        Verificar disponibilidade domínio no Registro.br
      </h1>
      <p
        class="Upload__description"
      >
        Importe uma lista de nomes em Excel para verificar se já possuem domínios registrados.
      </p>
      <form
        action="/api/upload"
        class="Upload__form"
        enctype="multiplart/form-data"
        method="POST"
      >
        <button
          class="Upload__form__btn"
          type="button"
        >
          Selecionar Arquivo Excel
        </button>
        <input
          accept="          .xlsx,          .xls,          application/vnd.openxmlformats-officedocument.spreadsheetml.sheet,          application/vnd.ms-excel"
          aria-hidden="true"
          class="Upload__form__btn__file"
          id="uploadFile"
          multiple=""
          type="file"
        />
      </form>
    </section>
  </main>
</div>
`;
